build.cmd

```diff
@@ -21,45 +25,76 @@ set "__SourceDir=%__ProjectDir%\src"
-if /i "%1" == "x86"    (set __BuildArch=x86&&shift&goto Arg_Loop)
-if /i "%1" == "arm"    (set __BuildArch=arm&&shift&goto Arg_Loop)
-if /i "%1" == "arm64"    (set __BuildArch=arm64&&shift&goto Arg_Loop)
-
-if /i "%1" == "debug"    (set __BuildType=Debug&shift&goto Arg_Loop)
-if /i "%1" == "release"   (set __BuildType=Release&shift&goto Arg_Loop)
-if /i "%1" == "checked"    (set __BuildType=Checked&shift&goto Arg_Loop)
-
-if /i "%1" == "clean"   (set __CleanBuild=1&shift&goto Arg_Loop)
-
-if /i "%1" == "freebsdmscorlib" (set __MscorlibOnly=1&set __BuildOS=FreeBSD&shift&goto Arg_Loop)
-if /i "%1" == "linuxmscorlib" (set __MscorlibOnly=1&set __BuildOS=Linux&shift&goto Arg_Loop)
-if /i "%1" == "osxmscorlib" (set __MscorlibOnly=1&set __BuildOS=OSX&shift&goto Arg_Loop)
-if /i "%1" == "windowsmscorlib" (set __MscorlibOnly=1&set __BuildOS=Windows_NT&shift&goto Arg_Loop)
-
-if /i "%1" == "vs2013" (set __VSVersion=%1&shift&goto Arg_Loop)
-if /i "%1" == "vs2015" (set __VSVersion=%1&shift&goto Arg_Loop)
-if /i "%1" == "skiptestbuild" (set __SkipTestBuild=1&shift&goto Arg_Loop)
-if /i "%1" == "docrossgen" (set __DoCrossgen=1&shift&goto Arg_Loop)
-
-if /i "%1" == "priority" (set __TestPriority=%2&shift&shift&goto Arg_Loop)
-
-if /i "%1" == "/toolset_dir" (set __ToolsetDir=%2&shift&shift&goto Arg_Loop)
 
-echo Invalid commandline argument: %1
+if /i "%1" == "/?"    goto Usage
+if /i "%1" == "-?"    goto Usage
+if /i "%1" == "/h"    goto Usage
+if /i "%1" == "-h"    goto Usage
+if /i "%1" == "/help" goto Usage
+if /i "%1" == "-help" goto Usage
+
+if /i "%1" == "x64"                 (set __BuildArch=x64&shift&goto Arg_Loop)
+if /i "%1" == "x86"                 (set __BuildArch=x86&shift&goto Arg_Loop)
+if /i "%1" == "arm"                 (set __BuildArch=arm&shift&goto Arg_Loop)
+if /i "%1" == "arm64"               (set __BuildArch=arm64&shift&goto Arg_Loop)
+
+if /i "%1" == "debug"               (set __BuildType=Debug&shift&goto Arg_Loop)
+if /i "%1" == "release"             (set __BuildType=Release&shift&goto Arg_Loop)
+if /i "%1" == "checked"             (set __BuildType=Checked&shift&goto Arg_Loop)
+
+if /i "%1" == "clean"               (set __CleanBuild=1&shift&goto Arg_Loop)
+
+if /i "%1" == "freebsdmscorlib"     (set __MscorlibOnly=1&set __BuildOS=FreeBSD&shift&goto Arg_Loop)
+if /i "%1" == "linuxmscorlib"       (set __MscorlibOnly=1&set __BuildOS=Linux&shift&goto Arg_Loop)
+if /i "%1" == "osxmscorlib"         (set __MscorlibOnly=1&set __BuildOS=OSX&shift&goto Arg_Loop)
+if /i "%1" == "windowsmscorlib"     (set __MscorlibOnly=1&set __BuildOS=Windows_NT&shift&goto Arg_Loop)
+
+if /i "%1" == "vs2013"              (set __VSVersion=%1&shift&goto Arg_Loop)
+if /i "%1" == "vs2015"              (set __VSVersion=%1&shift&goto Arg_Loop)
+if /i "%1" == "skipmscorlibbuild"   (set __SkipMscorlibBuild=1&shift&goto Arg_Loop)
+if /i "%1" == "skipnativebuild"     (set __SkipNativeBuild=1&shift&goto Arg_Loop)
```

Comment:
I could. Of course, it creates an asymmetry with "skiptestbuild" which also exists here in build.cmd. I could change it to "skiptests" like build.sh, and leave a "skiptestbuild" for backward compat. Then all the "skip" would be consistent. Thoughts?